django_statsd/patches/cache.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, cache):
     def __getattribute__(self, attr):
         if attr == 'cache':
             return BaseCache.__getattribute__(self, attr)
+        # Don't wrap default_timeout in a functools.partial object (see wrap)
+        # This ensures compatibility with django-parler 1.6 under Django 1.9
```

Comment:
Is it really specific to Django 1.9 ?

Reply:
I wasn't involved when this change to introduced. This issue came up when we migrated to Django 1.9. I think it is the combination of 1.9 and django-parler that conflicted with this.

Anyway, I don't think it will do any harm having it this way.